util/json.go

```diff
@@ -77,12 +77,55 @@ func MustMarshalJSON(x interface{}) []byte {
 // Thereby, it is converting its argument to the representation expected by
 // rego.Input and inmem's Write operations. Works with both references and
 // values.
-func RoundTrip(x *interface{}) error {
+func RoundTrip(x *interface{}) (interface{}, error) {
```

Comment:
Help me out here, please, I don't see right away why the previous interface, changing `x` in-place, isn't possible with the different approach.